source/instructors/setupcourses/assignment-settings/prerequisite-assignments.rst

```diff
@@ -34,6 +34,6 @@ The **Copy Workspace From** setting allows you to copy content from one assignme
 
 When the assignment is created by the student, the workspace files/folders (excluding **.guides**, **.git** folder, any node modules and any **.codio**, **.codio-menu** or **.settings** files) of the assignment selected in the dropdown will be copied into the prerequisite assignment's workspace and will be included in the new assignment. 
```

Comment:
Should it be:
"...of the assignment selected in the dropdown will be copied **from** the prerequisite assignment's workspace"?

Reply:
Looking at helper text for Prerequisite assignment where it says: When enabled, students will not be able to start this assignment until prerequisite conditions are met I'd say should be 'copied into'